eng/pipelines/test-unix-job-single-machine.yml

```diff
@@ -0,0 +1,71 @@
+# Test on Unix using a single machine
```

Comment:
This new file and the analogous new files are just copies of the master version of the files with little to no modification.